integration-test/run-it.sh

```diff
@@ -36,10 +36,13 @@ then
   chmod o+rx /home/travis
 fi
 list_storm_processes || true
-# increasing swap space so we can run lots of workers
-sudo dd if=/dev/zero of=/swapfile.img bs=4096 count=1M
-sudo mkswap /swapfile.img
-sudo swapon /swapfile.img
+if [ "$(uname -m)" != aarch64 ]; then
```

Comment:
Is this change still needed after the change to arm64 and xenial

Reply:
yes, there still have problems of swap support on Travis ARM64 resources, you can take a look the error: 
https://travis-ci.com/github/apache/storm/jobs/460842259

Reply:
Thanks for validating. 

For future reference, the log is 
```
+sudo dd if=/dev/zero of=/swapfile.img bs=4096 count=1M
1048576+0 records in
1048576+0 records out
4294967296 bytes (4.3 GB, 4.0 GiB) copied, 27.2215 s, 158 MB/s
+sudo mkswap /swapfile.img
Setting up swapspace version 1, size = 4 GiB (4294963200 bytes)
no label, UUID=ba655d1c-230e-4feb-a06a-e342c114d7eb
+sudo swapon /swapfile.img
swapon: /swapfile.img: insecure permissions 0644, 0600 suggested.
swapon: /swapfile.img: skipping - it appears to have holes.
The command "if [[ $(uname -m) != 's390x' ]]; then /bin/bash ./dev-tools/travis/travis-script.sh `pwd` $MODULES; fi" exited with 255.
```